codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+schema: "https://jabref-online.herokuapp.com/api"
+documents: "src/graphql/**/*.graphql"
+generates:
+  src/generated/graphql.tsx:
```

Comment:
I'm not completely sure but I think `d.ts` is a more common extension for type definition files, or not?

Reply:
Right